elements/lisk-p2p/src/peer/base.ts

```diff
@@ -182,7 +172,7 @@ export class Peer extends EventEmitter {
 				respond(error);
 				this.emit(EVENT_INVALID_REQUEST_RECEIVED, {
 					packet,
-					peerId: this._id,
+					peerId: this.peerInfo.peerId,
```

Comment:
this._peerInfo.peerId

Reply:
When the getters exist, we should make use of them to read data.